### ISSUE TITLE
[FEAT] Social networking web sites

### ISSUE THREAD

**Issue by photonepoch** [jerlendds/osintbuddy#35]
This is powerfull tool, but it needed support for social networking. 
Back in the day I did something not quite similar using dumb python scripts, for example to pull date from VK website for analysis, but you know, given that this tool have great potential to make truly open-source osint software, it should be incorporated.

From my share I would try to cook something up related to Russian-speaking SN web sites, see you soon.

**Comment by jerlendds** [comment]
I agree, I'd like to eventually add some social networks as well. I'm currently working on a huge refactor and migrating all the features over to a plugin architecture, once that's complete I'll be more active in adding features again and it will be much easier for anyone to create their own plugin based of the soon to be osintbuddy python/pypi package
